logstash-event.gemspec

```diff
@@ -9,20 +9,34 @@ Gem::Specification.new do |gem|
 
   gem.files = %w{
     lib/logstash-event.rb
+    lib/logstash/environment.rb
+    lib/logstash/errors.rb
     lib/logstash/event.rb
+    lib/logstash/java_integration.rb
+    lib/logstash/json.rb
     lib/logstash/namespace.rb
-    lib/logstash/util/fieldreference.rb
+    lib/logstash/timestamp.rb
+    lib/logstash/version.rb
     lib/logstash/util.rb
-    spec/event.rb
+    lib/logstash/util/accessors.rb
+    lib/logstash/util/fieldreference.rb
     LICENSE
   }
 
-  gem.test_files    = []
+  gem.test_files    = ["spec/core/event_spec.rb"]
   gem.name          = "logstash-event"
   gem.require_paths = ["lib"]
-  gem.version       = "1.2.02"
+  gem.version       = "1.3.0"
```

Comment:
@suyograo As it has been lagging for a long time, I think it could simply be a proof that the gem is not forgotten anymore and tested against logstash 1.5.x from now. 
Also I suppose the addition of metadata coud explain the switch of version intended by @jsvd